### ISSUE TITLE
Label system for Proposals to ease the process review by CFP members

### ISSUE THREAD

**Issue by guw** [nicmarti/cfp-devoxx#49]
It would be great if the app can support adding (and removing) private PC tags to talks. It's really helpful for organizing talks during review. We currently have over 700 proposals for US.

I know this is not a trivial feature to implement but I wanted to add the wish here.

**Comment by nicmarti** [comment]
That's a good idea
So a basic personal tag system and a search/filter feature ?

**Comment by guw** [comment]
I think it should not be personal but shared between the members of the content committee. I found them useful when reviewing talks together.

**Comment by aheritier** [comment]
yes good idea. private tags for the review, for exemple to mark the ones we have doubts, the ones we want to review with others, ..

**Comment by nicmarti** [comment]
Ok thanks for the clarification. I'll try to propose something but for FR only for the time being

**Comment by nicmarti** [comment]
Hi
There is now a Tag system on the Devoxx US and BE source code.
The code was merged to the core CFP but lacks a bit of documentation. Thus I kept this issue open.

**Comment by nicmarti** [comment]
FYI the Tag system for Proposal is not related to this Request. 

As a speaker I can now add 3 tags to my Proposal. This is different from what was asked here, where we want "label" like Github. 

Thus I will try to change the title of this issue for clarification.
Thanks
